mapper/orm-coordination-outbox-polling/pom.xml

```diff
@@ -35,6 +35,34 @@
             <groupId>org.apache.avro</groupId>
             <artifactId>avro</artifactId>
         </dependency>
+
+        <!-- This dependency is only here to solve Moditect errors.
+             For some reason, the jakarta.transaction module requires jakarta.interceptor,
+             but the jakarta.transaction-api Maven module doesn't depend on jakarta.interceptor-api,
+             and that doesn't play well with Moditect, which ends up throwing an exception:
+               Execution add-module-infos of goal org.moditect:moditect-maven-plugin:1.0.0.RC2:add-module-info failed: Module jakarta.interceptor not found, required by jakarta.transaction
+         -->
+        <dependency>
+            <groupId>jakarta.interceptor</groupId>
+            <artifactId>jakarta.interceptor-api</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.jakarta.interceptor-api}</version>
+            <scope>provided</scope>
+        </dependency>
+        <!-- This dependency is only here to solve Moditect errors.
+             For some reason, the jakarta.transaction module requires jakarta.cdi,
+             but the jakarta.transaction-api Maven module doesn't depend on jakarta.enterprise.cdi-api,
+             and that doesn't play well with Moditect, which ends up throwing an exception:
+               Execution add-module-infos of goal org.moditect:moditect-maven-plugin:1.0.0.RC2:add-module-info failed: Module jakarta.interceptor not found, required by jakarta.transaction
+         -->
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <!-- DO NOT REMOVE and DO NOT MANAGE the version of this dependency. See the version property declaration. -->
+            <version>${version.jakarta.enterprise}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Did you check whether this workaround is still necessary? We've upgraded these dependencies multiple times since the workaround was first introduced.

Reply:
`jakarta.enterprise.cdi-api` is still needed, but `jakarta.interceptor-api` can be removed same as in the other case, but  `jakarta.xml.bind-api` is additionally required. I think it was complaining about this one when we had an xml-bind dependency in orm mapper (the one to address exclusions)